### ISSUE TITLE
Question about GuidedMixup

### ISSUE THREAD

**Issue by khawar-islam** [3neutronstar/GuidedMixup#1]
Dear @3neutronstar ,

I hope this email finds you well. I have a question regarding your GuidedMixup paper. How did you split the Caltech-101 dataset? Are there specific training and testing images?

I would appreciate your prompt response as I need to conduct an experiment urgently.

Best regards,

**Comment by 3neutronstar** [comment]
Dear @khawar-islam ,

Thank you for being so interested in my work. 

Yes, there are no specific training and testing images in the official.
To this end, I split the Caltech-101 dataset into 0.8:0.2, following the scikit-learn package's `train_test_split` function. I also use the `stratify` option using the labels.

**Comment by khawar-islam** [comment]
Dear @3neutronstar 

thank you for your message. The Caltech-101 folder contains 102 folders  not 101 folders. What is the reason behind this?

**Comment by 3neutronstar** [comment]
I use Torchvision's Caltech-101 class to train the model. I think it automatically ignores the folder name `BACKGROUND_Google` in 101_ObjectCategories directory.

**Comment by khawar-islam** [comment]
@3neutronstar thanks a lot this is a lot for me

**Comment by khawar-islam** [comment]
Dear @3neutronstar Would it be possible for you to give Aircraft and Caltech training file?
